.github/workflows/acceptance.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   frontend:
     name: frontend tests
     runs-on: ubuntu-16.04
-    timeout-minutes: 20
+    timeout-minutes: 25
```

Comment:
Let's not increase the timeout past 20 minutes, this should be the upper limit (with the dev-env test as an exception).

Reply:
I think this is a temporary change to accomodate the cache miss?

Reply:
I bumped it because I hit a timeout once. I've removed it. Let's see if it happens again.